src/Ccap/Codegen/Purescript.purs

```diff
@@ -57,16 +56,12 @@ mergeImports :: Array PsImport -> Array PsImport
 mergeImports imps =
   let
     sorted = Array.sortBy ((compare `on` _.mod) <> (compare `on` _.alias)) imps
-    grouped = Array.groupBy (\a b -> a.mod == b.mod && b.alias == b.alias) sorted
-  in grouped <#> \group -> do
-    let typs = map _.typ group
-    { mod: (NonEmptyArray.head group).mod
-    , alias: (NonEmptyArray.head group).alias
-    , typ:
-        if any isNothing typs
-          then Nothing
-          else Just $ intercalate ", " (Array.nub <<< compact <<< NonEmptyArray.toArray $ typs)
-    }
+    grouped = Array.groupBy (\a b -> a.mod == b.mod && a.alias == b.alias) sorted
+  in grouped <#> \group ->
+    (NonEmptyArray.head group)
+      { typ = traverse _.typ group <#> NonEmptyArray.toArray
+                >>> Array.sort >>> Array.nub >>> intercalate ", "
+      }
```

Comment:
If we didn't need to worry about inconsistent aliases, this could probably be simplified back to something like https://github.com/ccap/purescript-ccap-codegen/pull/20/commits/4d67ecb7161df22e7aba3f32f74d51251d958096#diff-a2d0fa98f8aa6dfd7a52cc9d690ddc2bL37